test/lic0Test.java

```diff
@@ -0,0 +1,64 @@
+import decide.Connectors;
+import decide.Coordinate;
+import decide.Decider;
+import decide.Parameters;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class lic0Test {
```

Comment:
Test names include no description of what is tested, which makes them difficult to read/understand.